index.html

```diff
@@ -6,36 +6,26 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./assets/css/styles.css">
-    <title>First Project</title>
+    <title>Javier Martinez's Portfolio</title>
 </head>
 
 <body>
     <header>
-        <div class="topHeader">
-            <div class="logo">
-                <a href="#" rel="noopener" target="_blank">
-                    My Logo
-                </a>
-            </div>
-            <nav>
-                <ul class="links-index">
-                    <li>
-                        <a href="#">Portfolio</a>
-                    </li>
-                    <li>
-                        <a href="#">About</a>
-                    </li>
-                    <li>
-                        <a href="#">Contact</a>
-                    </li>
-                </ul>
-                <div id="breadcrumb">
-                    <a href="#" rel="noopener" target="_blank">
-                        <img src="./assets/images/breadcrumb.png" alt="">
-                    </a>
-                </div>
-            </nav>
+        <div class="logo">
+          <a href="#">My Logo</a>
         </div>
+  
+        <nav>
+          <div class="hamburger">
+            <img src="./assets/images/breadcrumb.png" alt="" />
+          </div>
+          <div class="main-menu">
+            <span class="close">X</span>
```

Comment:
- [ ] [OPTIONAL] I suggest that you make the `hover` effect of the X-Icon `cursor` as you've done for the other links. This is good for semantics an UX reason.

Reply:
Thanks for this optional suggestion, my brother. I'm running out of time 3.50 am. here ... and two projects to do before the end of the week. I'll kindly take that optional suggestion to implement it on my weekend, dear @AdnanOlarmmi